tests/data/MethodBarEWithoutReturnType.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/*
+ * Copyright (c) Since 2007 PrestaShop.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Space;
+
+class E
+{
+    public function bar()
```

Comment:
This class should be refused with the Rule, but it was excluded in configuration from the analysis.

Reply:
So it's the usecase "this should rejected by PHPstan but as it was already there before and we dont want to introduce a BC break we ignore it"